app/seed-data/formula_values/data_processed/EFDB_2006_IPCC_guidelines/Methodology.csv

```diff
@@ -11,3 +11,4 @@ e4508774-ab32-3827-aa3a-8cdf9d943631,incineration-waste-inboundary-methodology,,
 b0da6b28-c5e0-3f8a-859f-96bac5ff7f6c,wastewater-outside-domestic-calculator-activity,,6a508faa-80a8-3246-9941-90d8cc8dec85
 a6c1fa15-4405-3900-b649-966c8c7aeae6,wastewater-inside-industrial-calculator-activity,,6a508faa-80a8-3246-9941-90d8cc8dec85
 946471de-43e5-39f4-ae8a-befa344bfaee,wastewater-outside-industrial-calculator-activity,,6a508faa-80a8-3246-9941-90d8cc8dec85
+3b151e3f-c71a-3fb6-935e-7cce853055d3,wastewater-outside-industrial-calculator-activity,,6a508faa-80a8-3246-9941-90d8cc8dec85

```

Comment:
This is a temp fix, let the data team know that the entry for this UUID is missing